library/ssl_ciphersuites.c

```diff
@@ -293,7 +293,7 @@ static const mbedtls_ssl_ciphersuite_t ciphersuite_definitions[] =
 {
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3)
 #if defined(MBEDTLS_AES_C)
-#if defined(MBEDTLS_GCM_C)
+#if defined(MBEDTLS_CIPHER_HAVE_GCM)
```

Comment:
Intuitively I'm inclined to use TLS specific macros for TLS, even though right now I'm not sure I can fully justify why. Probably needs more thinking/discussion.